### ISSUE TITLE
[i] SC 1.3.5 Script - Scene 5 Audio

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-wcag-videos#31]
Strong
To me it seems like the point of this video is to show how input fields that are very common among different websites should use the same proper naming markup so that if a person saves data in their browser for a specific field then it will be pulled up and used in other websites to save on typing it all out again. If this is accurate, then I think the suggested change below is needed. Or something similar. I know it is a bit longer, but I feel it is more clear to a developer what is happening.

Current:

> She allows her username and email address to be stored for her school website because they are used frequently. When these are requested, they are automatically filled by the browser and it saves Preety a lot of time and effort.

Suggestion:

> **_On her school website_** she allows her username and email address to be stored because they are used frequently. When **_this information is_** requested **_on other sites, it is_** automatically filled **_in_** by the browser  and it saves Preety a lot of time and effort.

**Comment by nitedog** [comment]
Good question, I'm not sure if this is common behavior. I think my browser stores some types of inputs, like email address, but doesn't automatically fill in -- I can select from prior inputs when I get to the particular field.

**Comment by bakkenb** [comment]
You are correct @nitedog, it is not automatically filled in. I guess the original script was not too accurate then. In my browser it is "suggested" in the field and you can either type something else or hit the return key and it will add the suggested text to that field (and sometimes to a bunch of other fields).

What about:

> On her school website she allows her username and email address to be stored because they are used frequently. When this information is requested on other sites, the field suggested previous information that can be selected and it saves Preety a lot of time and effort.  (a bit wordy - maybe others have better edits)

**Comment by nitedog** [comment]
Fairly extensive rewrite to address this issue: see [preview](https://deploy-preview-42--wai-wcag-videos.netlify.app/sc-1.3.5/) and [diff](https://github.com/w3c/wai-wcag-videos/pull/42/files)
